server/fleet/activities.go

```diff
@@ -876,8 +877,10 @@ func (a ActivityTypeFleetEnrolled) ActivityName() string {
 func (a ActivityTypeFleetEnrolled) Documentation() (activity string, details string, detailsExample string) {
 	return `Generated when a host is enrolled to Fleet (Fleet's agent fleetd is installed).`,
 		`This activity contains the following fields:
+- "host_id": ID of the host.
 - "host_serial": Serial number of the host.
 - "host_display_name": Display name of the host.`, `{
+	"host_id": "123",
```

Comment:
Just a heads up, this is a mix of spaces/tabs

Reply:
Indeed! Just followed the pattern for the existing 2 fields there. Important to change?

Reply:
updated 👍